### PR TITLE
Update stato.md

### PR DESCRIPTION
**Pull request by proccaserra**
adding publications 
adding usage examples

### CODE REVIEW

File: ontology/stato.md [hunk 1]
```diff
@@ -18,10 +18,40 @@ license:
...
+  - id: https://doi.org/10.1016/j.jbi.2021.103685
+    title: 'Making science computable: Developing code systems for statistics, study design, and risk of bias'
+  type: annotation
+  user: https://fevir.net/resources/CodeSystem/27270#STATO:0000039
+- description: PSI-MS CV
+  publications:
+  - id: https://doi.org/10.1093/database/bat009
+    title: The HUPO proteomics standards initiative- mass spectrometry controlled vocabulary.
+  type: annotation
+  user: https://github.com/HUPO-PSI/psi-ms-CV
```
Comment (cmungall) [inline ontology/stato.md:48]: is this true? It looks like they recently dropped STATO (for some easily fixable technical reason?)

**Comment by nlharris** [comment]
Can this be squashed & merged?

File: ontology/stato.md [hunk 2]
```diff
@@ -18,10 +18,34 @@ license:
 preferredPrefix: STATO
 products:
 - id: stato.owl
+publications:
+- id: https://doi.org/10.1038/s41597-019-0286-0
```
Comment (deepakunni3) [inline ontology/stato.md:22]:
```suggestion
- id: https://pubmed.ncbi.nlm.nih.gov/31831744
```

File: ontology/stato.md [hunk 3]
```diff
@@ -18,10 +18,34 @@ license:
...
+  title: Experiment design driven FAIRification of omics data matrices, an exemplar
+- id: https://doi.org/10.1038/s41597-020-0409-7
```
Comment (deepakunni3) [inline ontology/stato.md:24]:
```suggestion
- id: https://pubmed.ncbi.nlm.nih.gov/32109232
```

File: ontology/stato.md [hunk 4]
```diff
@@ -18,10 +18,34 @@ license:
...
+  title: Semantic concept schema of the linear mixed model of experimental observations
 repository: https://github.com/ISA-tools/stato
 tags:
 - statistics
 tracker: https://github.com/ISA-tools/stato/issues
+usages:
+- description: struct (Statistics in R using Class-based Templates), Struct integrates with the STATistics Ontology to ensure consistent reporting and maximizes semantic interoperability
+  publications:
+  - id: https://doi.org/10.1101/2021.10.10.463703
```
Comment (deepakunni3) [inline ontology/stato.md:33]: There is a mismatch between the publication DOI and the title. The DOI refers to a paper that has the title 'Brain Data Standards - A method for building data-driven cell-type ontologies' where as the 'title' field is referring to 'struct: an R/Bioconductor-based framework for standardized metabolomics data analysis and beyond'.

Did you mean to use https://pubmed.ncbi.nlm.nih.gov/33325493 instead?

File: ontology/stato.md [hunk 5]
```diff
@@ -18,10 +18,34 @@ license:
...
+    title: 'struct: an R/Bioconductor-based framework for standardized metabolomics data analysis and beyond'
+  type: annotation
+  user: https://bioconductor.org/packages/release/bioc/html/struct.html
+- description: Scientific Evidence Code System (SEVCO) on the FEvIR platform. The FEvIR Platform includes many Builder Tools to create FHIR Resources without requiring expertise in FHIR or JSON, and Converter Tools to convert structured data to FHIR Resources
+  publications:
+  - id: https://doi.org/10.1016/j.jbi.2021.103685
```
Comment (deepakunni3) [inline ontology/stato.md:39]:
```suggestion
  - id: https://pubmed.ncbi.nlm.nih.gov/33486066
```

File: ontology/stato.md [hunk 6]
```diff
@@ -18,10 +18,34 @@ license:
...
+    title: 'Making science computable: Developing code systems for statistics, study design, and risk of bias'
+  type: annotation
+  user: https://fevir.net/resources/CodeSystem/27270#STATO:0000039
+- description: OBCS
+  publications:
+  - id: https://doi.org/10.1186/s13326-016-0100-2
```
Comment (deepakunni3) [inline ontology/stato.md:45]:
```suggestion
  - id: https://pubmed.ncbi.nlm.nih.gov/27627881
```

File: ontology/stato.md (hunk 2 above)
Comment (deepakunni3) [reply ontology/stato.md:22]: Favoring PMID over DOI.

File: ontology/stato.md (hunk 5 above)
Comment (deepakunni3) [reply ontology/stato.md:39]: Favoring PMID over DOI.

File: ontology/stato.md (hunk 6 above)
Comment (deepakunni3) [reply ontology/stato.md:45]: Favoring PMID over DOI.

File: ontology/stato.md (hunk 3 above)
Comment (deepakunni3) [reply ontology/stato.md:24]: Favoring PMID over DOI.

**Review by deepakunni3 — CHANGES_REQUESTED** [review]

**Comment by wdduncan** [comment]
Hi @proccaserra.  Can you please review @deepakunni3 suggestions.

File: ontology/stato.md (hunk 6 above)
Comment (proccaserra) [reply ontology/stato.md:45]: thank you @deepakunni3

File: ontology/stato.md (hunk 4 above)
Comment (proccaserra) [reply ontology/stato.md:33]: indeed !

File: ontology/stato.md [hunk 7]
```diff
@@ -18,10 +18,34 @@ license:
...
+- id: https://doi.org/10.1038/s41597-020-0409-7
+  title: Semantic concept schema of the linear mixed model of experimental observations
 repository: https://github.com/ISA-tools/stato
 tags:
 - statistics
 tracker: https://github.com/ISA-tools/stato/issues
+usages:
+- description: struct (Statistics in R using Class-based Templates), Struct integrates with the STATistics Ontology to ensure consistent reporting and maximizes semantic interoperability
+  publications:
+  - id: https://doi.org/10.1101/2021.10.10.463703
```
Comment (proccaserra) [inline ontology/stato.md:33]:
```suggestion
  - id: https://pubmed.ncbi.nlm.nih.gov/33325493
```

File: ontology/stato.md [hunk 8]
```diff
@@ -18,10 +18,34 @@ license:
...
+  user: https://bioconductor.org/packages/release/bioc/html/struct.html
+- description: Scientific Evidence Code System (SEVCO) on the FEvIR platform. The FEvIR Platform includes many Builder Tools to create FHIR Resources without requiring expertise in FHIR or JSON, and Converter Tools to convert structured data to FHIR Resources
+  publications:
+  - id: https://pubmed.ncbi.nlm.nih.gov/33486066
+    title: 'Making science computable: Developing code systems for statistics, study design, and risk of bias'
+  type: annotation
+  user: https://fevir.net/resources/CodeSystem/27270#STATO:0000039
+- description: OBCS
+  publications:
+  - id: https://doi.org/10.1186/s13326-016-0100-2
```
Comment (proccaserra) [inline ontology/stato.md:45]:
```suggestion
  - id: https://pubmed.ncbi.nlm.nih.gov/27627881
```

File: ontology/stato.md [hunk 9]
```diff
@@ -18,10 +18,34 @@ license:
 preferredPrefix: STATO
 products:
 - id: stato.owl
+publications:
+- id: https://pubmed.ncbi.nlm.nih.gov/31831744
+  title: Experiment design driven FAIRification of omics data matrices, an exemplar
+- id: https://doi.org/10.1038/s41597-020-0409-7
```
Comment (proccaserra) [inline ontology/stato.md:24]:
```suggestion
- id: Semantic concept schema of the linear mixed model of experimental observations
```

**Review by deepakunni3 — APPROVED** [review]

**Pull request merged**